File: ProjectTwo(Python)/models/request_credit.py

class RequestCredit:

    def __init__(self, req_id=0, user_id=0, card_type="", credit_limit=0, req_status=""):
        self.req_id = req_id
        self.user_id = user_id
        self.card_type = card_type
        self.credit_limit = credit_limit
        self.req_status = req_status

    def __repr__(self):
        return str({
            'req_id': self.req_id,
            'user_id': self.user_id,
            'card_type': self.card_type,
            'credit_limit': self.credit_limit,
            'req_status': self.req_status
        })

    def json(self):
        return str({
            'reqId': self.req_id,
            'userId': self.user_id,
            'cardType': self.card_type,
            'creditLimit': self.credit_limit,
            'reqStatus': self.req_status
        })

File: JavaBackend/src/main/java/models/RequestCredit.java
package models;

public class RequestCredit {

    private int reqID;
    private int userID;
    private String cardType;
    private double creditLimit;
    private String reqStatus;

    public RequestCredit(){}

    public RequestCredit(int reqID, int userID, String cardType, double creditLimit, String reqStatus){
        this.reqID=reqID;
        this.userID=userID;
        this.cardType=cardType;
        this.creditLimit=creditLimit;
        this.reqStatus=reqStatus;
    }

    public int getReqID() {
        return reqID;
    }

    public void setReqID(int reqID) {
        this.reqID = reqID;
    }

    public int getUserID() {
        return userID;
    }

    public void setUserID(int userID) {
        this.userID = userID;
    }

    public String getCardType() {
        return cardType;
    }

    public void setCardType(String cardType) {
        this.cardType = cardType;
    }

    public double getCreditLimit() {
        return creditLimit;
    }

    public void setCreditLimit(double creditLimit) {
        this.creditLimit = creditLimit;
    }

    public String getReqStatus() {
        return reqStatus;
    }

    public void setReqStatus(String reqStatus) {
        this.reqStatus = reqStatus;
    }
}
